packages/app-cli/tests/support/pluginRepo/README.md

```diff
@@ -15,7 +15,7 @@ This repository contains the following plugins:
 --- | --- | --- | --- | ---
 [🏠](https://discourse.joplinapp.org/t/insert-referencing-notes-backlinks-plugin/13632) | Backlinks to note | 1.0.4 | Creates backlinks to opened note | a  
 [🏠](https://github.com/JackGruber/joplin-plugin-combine-notes) | Combine notes | 0.2.1 | Combine one or more notes | JackGruber
-[🏠](https://github.com/JackGruber/joplin-plugin-copytags) | Copy Tags | 0.3.2 | Plugin to extend the Joplin tagging menu with a coppy all tags and tagging list with more control. | JackGruber
+[🏠](https://github.com/JackGruber/joplin-plugin-copytags) | Copy Tags | 0.3.2 | Plugin to extend the Joplin tagging menu with a copy all tags and tagging list with more control. | JackGruber
```

Comment:
Did you check the corresponding test?

e.g. you can't just change text in a test, because the test could now fail or pass (when it passed or failed before)